### PR TITLE
(maint) Add docs for 6.11.0 release

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

File: documentation/release_notes.markdown [hunk 1]
```diff
@@ -10,6 +10,14 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.11.0
+
+Released 30 April 2020
+
+### New features
+
+- The `puppetserver ca` CLI tool has been updated to version 1.7.0. It will now show any authorization extensions that exist when listing certificates or csrs.
```
Comment (Magisus) [inline documentation/release_notes.markdown:19]: Do we usually capitalize "CSRs"?

Comment (justinstoller) [reply documentation/release_notes.markdown:19]: Good question. It looks like there's only one other spot we mention them and it's capitalized. I'll update.

**Comment by justinstoller** [comment]
Note: We want approval from Froyo and then from @clairecadman but we won't want to merge until after the go/no-go decision tomorrow (since merged changes are auto-deployed to the docs site at a regular interval).

**Comment by clairecadman** [comment]
@justinstoller Same as the other notes re. missing Jira ticket. But other than that they look good! Thanks!

**Comment by justinstoller** [comment]
Updated w/ ticket link. Feel free to merge now!

**Pull request merged**